back/.rubocop.yml

```diff
@@ -20,20 +20,22 @@ Metrics/BlockLength:
   - 'db/migrate/**/*'
   # Max: 25
 Metrics/MethodLength:
-  IgnoredMethods: ['included', 'prepended', 'extended']
   Enabled: true
+  Max: 50
+  IgnoredMethods: ['included', 'prepended', 'extended']
   Exclude:
   - 'db/migrate/**/*'
-  # Max: 10
 Metrics/AbcSize:
+  Enabled: false
```

Comment:
Maybe I would leave it enabled as a good reminder that we have to write simple methods. Maybe increasing `Max` value.

Reply:
Sure, I'll turn it back on, since I forgot for which part of the code I did it. Personally, I find the ABC metric a bit too complicated and feel like method length etc. might be sufficient.